### DISCUSSION TITLE
Adding a Driverless AI model to H2O Wave

### DISCUSSION THREAD

**Discussion by GeorgiaJR** [h2oai/wave#1109] [Q&A]
Hello.

I am currently running models on H2O Driverless AI and I want to be able to export the final model and load this into the H2O Wave app that I have running locally on my machine.

I have tried to follow the WaveML tutorials [here](https://wave.h2o.ai/docs/examples/ml-dai) but these are to build and train models and also require the Wave application to run on the cloud and through the app store. 

Is there any way I can export a trained model from Driverless AI and deploy this onto H2O Wave? I am having difficulty finding anything online to explain how to do this. 

Thanks in advance!

**Comment by sairaamv** [comment]
Hi @GeorgiaJR ,

Its a bit open-ended but will give my shot to answer. There are a couple of ways to connect Driverless AI models with H2O Wave. 

1. Using H2O AI Cloud as you mentioned
2. Using `pip install driverlessai` and connect your driverless-ai instance
3. Download Driverless AI model artefacts and upload/connect with H2O Wave


To my exposure, I agree that there is not a straight-up boilerplate available to have these two modules together in a public git repo. I would recommend exploring in automating/downloading model artefacts from driverlessai connector (Path-2) and explore an app which you could modify from any of these available example [here](https://github.com/h2oai/wave-apps). 

If possible, put these modules together based on your quick-need and I will definitely PR your repo there.
